Frontend/ui-customer/src/Services/Backend.purs

```diff
@@ -19,7 +19,7 @@ import Services.API
 
 import Control.Monad.Except.Trans (lift)
 import Control.Transformers.Back.Trans (BackT(..), FailBack(..))
-import Common.Types.App (Version(..), SignatureAuthData(..), LazyCheck (..))
+import Common.Types.App (Version(..), SignatureAuthData(..), LazyCheck (..), FeedbackAnswer)
```

Comment:
Can you remove this unused import - `FeedbackAnswer`

Reply:
we using this `FeedbackAnswer` in `makeRideFeedBackReq` function